src/TritonRoute/src/dr/FlexDR_init.cpp

```diff
@@ -963,6 +976,8 @@ void FlexDRWorker::initNet_termGenAp_new(const frDesign* design, drPin* dPin)
           if (!boost::polygon::intersect(pinRect, routeRect)) {
             continue;
           }
+          bool noJog = getTech()->getLayer(currLayerNum)->
+                        getLef58RightWayOnGridOnlyConstraint() != nullptr;
```

Comment:
use ->isUnidirectional() instead

Reply:
the return value of this function is different: 

getNumMasks() > 1 || getLef58RectOnlyConstraint()

It may get different results. Note that this is just a quick fix so we can merge this branch soon and get Louii's updates. A proper solution (removal of on-the-fly aps) is on its way, so I dont think we should be so much worried about this function (initNet_termGenAp_new).

Reply:
ok